target/linux/lantiq/files-4.19/arch/mips/boot/dts/lantiq/danube_arcadyan_arv752dpw.dts

```diff
@@ -157,6 +159,7 @@
 &gsw {
 	phy-mode = "rmii";
 	mtd-mac-address = <&boardconfig 0x16>;
+	mtd-mac-address-increment = <1>;
```

Comment:
I'm fine with adding the switch configuration as well as using a unique MAC address for the WiFi interface.

Regarding the LAN mac address assignment I'm with @adrianschmutzler in this matter. The LAN and Wireless MAC are identical in the vendor firmware. At least this seems to be the case with the similar EasyBox 803 (see picture 5 of this [post ](https://www.computerbase.de/forum/threads/repeater-einrichten.1818328/#post-21609934)).

We should try to resemble the stock behavior where this is feasible. This is obviously the case here.

Reply:
Ok, the vendor is always right. ;-) Removed the proposed increment on ethernet mac as requested.